tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -123,6 +123,24 @@ public interface ICollectionItem
                 .x(() => fakes.Should().OnlyContain(item => Fake.GetFakeManager(item) != null));
         }
 
+        [Scenario]
+        public void InterfaceWithAlikeGenericMethod()
+        {
+            "Given an interface with methods having multiple generic arguments that can be inheirited"
+                .See<IInterfaceWithSimilarMethods>();
+
+            "It should create fake instance of this"

```

Comment:
We tend to follow Given/When/Then, so the "It" is jarring. "When I create a fake of the interface"?